packages/bootstrap/scss/scrollview/_theme.scss

```diff
@@ -1,35 +1,30 @@
 @include exports("scrollview/theme") {
 
-    .k-scrollview {}
+    //Remove these scoping selectors when the Angular team change the ScrollView rendering
```

Comment:
Log an issue in `kendo-angular-scrollview` repo and link it here in the comment - for reference: https://github.com/telerik/kendo-themes/blob/develop/packages/default/scss/notification/_layout.scss#L66